Umweltzone/src/main/res/values-es/strings.xml

```diff
@@ -0,0 +1,375 @@
+    <string name="city_info_section_title_dpz">Detalles sobre las zonas de prohibición de diésel</string>
+    <string name="city_info_section_title_other">Further information</string>
+
+    <string name="city_info_zone_status">Libre</string>
+    <string name="city_info_zone_number_as_of">
+        A partir del <xliff:g example="1 de Oct del 2099" id="date">%1$s</xliff:g> sólo los
+        vehiculos con etiqueta de color <xliff:g example="verde" id="with_colors">%2$s</xliff:g>
+        pueden ingresar a la zona de bajas emisiones.
+    </string>
+    <string name="city_info_zone_number_since">
+        Desde el <xliff:g example="1 de Feb del 2016" id="date">%1$s</xliff:g> sólo los
+        vehiculos con etiqueta de color <xliff:g example="verde" id="with_colors">%2$s</xliff:g>
+        pueden ingresar a la zona de bajas emisiones.
+    </string>
+    <string name="city_info_zone_number_since_date_format">d &#8217;de&#8217; MMMM &#8217;del&#8217; yyyy</string>
+    <string name="city_info_zone_number_since_text_fragment_green">
+        verde
+    </string>
+    <string name="city_info_zone_number_since_text_fragment_yellow">
+        amarillo o verde
+    </string>
+    <string name="city_info_zone_number_since_text_fragment_red">
+        rojo, amarillo o verde
+    </string>
+    <string name="city_info_zone_number_none">
+        Los vehículos de todas las clases de emisiones pueden entrar en la zona de bajas emisiones.
+    </string>
+    <string name="city_info_next_zone_number_as_of">
+         A partir del <xliff:g example="1 de Oct del 2099" id="date">%1$s</xliff:g> sólo los
+        vehiculos con etiqueta de color <xliff:g example="verde" id="with_colors">%2$s</xliff:g>
+        pueden ingresar a la zona de bajas emisiones.
+    </string>
+    <string name="city_info_abroad_licensed_vehicle_zone_info">
+        Los vehículos registrados en el extranjero pueden ingresar a la zona de bajas emisiones
+        hasta el
+        <xliff:g example="1 de Ene del 2016" id="date">%1$s</xliff:g>
+        con etiqueta de color
+        <xliff:g example="amarillo o verde" id="with_colors">%2$s</xliff:g>.
+    </string>
+
+    <string name="city_info_list_of_cities">
+        La zona contiene las siguientes ciudades o ciertos distritos:
+        <xliff:g example="Bochum, Castrop-Rauxel, Essen" id="cities">%1$s</xliff:g> and
+        <xliff:g example="Recklinghausen" id="last_city">%2$s</xliff:g>.
+    </string>
+
+    <string name="city_info_further_information">Más información en línea</string>
+    <string name="city_info_badge_online_title">Formulario de solicitud de etiquetas en línea</string>
+
+    <string name="city_info_geometry_source_text">Fuente de las coordenadas geométricas:
+        <xliff:g example="Geoportal Berlin / Umweltzone Berlin" id="geometrySource">%1$s</xliff:g>
+    </string>
+
+    <string name="city_info_geometry_updated_at_date_format">
+       MMMM d &#8217;del&#8217; yyyy
```

Comment:
This crashes the app because this must be a formatting string which defines the country-specific date formatting. Legal characters are `d` for day, `M` for month and `Y` for year. See also: [Java 8 documentation](https://docs.oracle.com/javase/8/docs/api/java/text/SimpleDateFormat.html).

Reply:
I try to escape some words, but is not working.

Reply:
`MMMM d \'del\' yyyy` works. Thanks.